CMakeLists.txt

```diff
@@ -643,6 +639,12 @@ if (WIN32 OR is_xcode OR unity)
 
   list(APPEND src ${beast_unity_srcs} ${ripple_unity_srcs})
 
+  add_with_props(src/unity/test_unity.cpp
```

Comment:
Add "per-source-file-properties" to the document outlining our complications with the build system